src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DocumentSynchronization/DocumentDidSaveEndpoint.cs

```diff
@@ -22,7 +23,7 @@ public TextDocumentIdentifier GetTextDocumentIdentifier(DidSaveTextDocumentParam
 
     public Task HandleNotificationAsync(DidSaveTextDocumentParams request, RazorRequestContext requestContext, CancellationToken cancellationToken)
     {
-        requestContext.Logger.LogInformation($"Saved Document {request.TextDocument.Uri.GetAbsoluteOrUNCPath()}");
+        requestContext.Logger.LogInformation("Saved Document {document}", request.TextDocument.Uri.GetAbsoluteOrUNCPath());
```

Comment:
Same comment here about ILogger already being a property/field of the endpoint class and then not using requestContext.Logger to log information.

Reply:
https://github.com/dotnet/razor/pull/9751